cmd/res/profiles/ED_ID1.yaml

```
@@ -0,0 +1,114 @@
+name: "ED_ID1"
```

Also change the file name accordingly 
```suggestion
name: "sample-2way-rest-device"
```